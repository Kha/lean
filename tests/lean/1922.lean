set_option pp.implicit true

structure C :=
( d : Π { X : Type }, list X → list X )

def P(c : C):= c.d [0]

#print P
