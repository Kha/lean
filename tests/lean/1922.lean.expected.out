def P : C → list ℕ :=
λ (c : C), @C.d c ℕ [0]
